doc/contributing/submitting-changes.chapter.md

```diff
@@ -82,6 +82,50 @@ If a security fix applies to both master and a stable release then, similar to r
 
 Critical security fixes may by-pass the staging branches and be delivered directly to release branches such as `master` and `release-*`.
 
+## Deprecating/removing packages {#submitting-changes-deprecating-packages}
+
+When software projects got discontinued, are unmaintained for a year, broken for a year or have severe security vulnerabilities that don't get fixed, we should remove them. Another reason would be that there is no maintainer for the package in nixpkgs for more than 6 months. In any case, it should always be attempted to find a new maintainer or fix smaller issues first.
```

Comment:
> When software projects got discontinued, are unmaintained for a year, broken for a year or have severe security vulnerabilities that don't get fixed, we should remove them. Another reason would be that there is no maintainer for the package in nixpkgs for more than 6 months.

This is quite a step and controversial. Probably better to leave the "when to remove" and stick with "how to remove" in this PR.

Reply:
How can we come to a policy here? Discuss in Forum or does it need an RFC?